### PR TITLE
fix: update picker component roles

### LINKED ISSUE #6884: fix: update picker item and picker list roles
# 🐛 Bug Report

With Picker presenting a list of selectable items selected using `aria-selected`, the roles for the picker list and the options should be `listbox` and `option` rather than `list` and `listitem` respectively.

## 🤔 Expected Behavior

The roles for the list and items should use appropriate roles for using aria-selected

## 😯 Current Behavior

The current roles used are `list` and `listitem` which tend to be more appropriate for static lists.

## 💁 Possible Solution

The roles that should be used are `listbox` and `option`.

## 🔦 Context

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

### PR DESCRIPTION
**Pull request by JiginJayaprakash**
Fixes #6884

# Pull Request

## 📖 Description

The roles are updated to listbox and option from list and listitem

### 🎫 Issues

(https://github.com/microsoft/fast/issues/6884)

### General

- [x] I have included a change request file using `$ yarn change`
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Comment by JiginJayaprakash** [comment]
@microsoft-github-policy-service agree

**Comment by scomea** [comment]
I agree with the menu and the menu items within changing.  Still not sure about the list of already selected items (pickerlist and pickerlistItem).  Let's discuss in the bug.

File: change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Fixes #6884",
```
Comment (radium-v) [inline change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json:3]: This comment field is used to generate changelog entries in [CHANGELOG.md](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/CHANGELOG.md). It defaults to the chosen commit message, but a more descriptive comment would be useful here:

```suggestion
  "comment": "update picker component roles",
```

File: change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```
Comment (radium-v) [inline change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json:2]: I would consider ARIA roles in a template to be a non-trivial change - the change commit might need to be regenerated to reflect that if others agree.

**Review by KingOfTac — DISMISSED** [review]
> I agree with the menu and the menu items within changing. Still not sure about the list of already selected items (pickerlist and pickerlistItem). Let's discuss in the bug.

I agree, I think for now we can just change the roles on the menu and menu items. The selected items will need further investigation.

@JiginJayaprakash Can you revert the role changes to pickerlist and pickerlistitem? Once that is done and @radium-v's comments are addressed on the change file I will feel good about approving this change.

File: change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json (hunk 2 above)
Comment (KingOfTac) [reply change/@microsoft-fast-foundation-35ae3cfb-b903-411c-b6fc-db23fb32474e.json:2]: I think type would still be prerelease.

**Review by scomea — APPROVED** [review]

**Review by JiginJayaprakash — COMMENTED** [review]
@KingOfTac  made suggested changes

File: change/@microsoft-fast-foundation-b31b1927-65a3-4498-b3c8-721295872198.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "update picker component roles",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "145117767+JiginJayaprakash@users.noreply.github.com",
+  "dependentChangeType": "patch"
```
Comment (KingOfTac) [inline change/@microsoft-fast-foundation-b31b1927-65a3-4498-b3c8-721295872198.json:6]: dependentChangeType should also be prerelease

**Review by bheston — APPROVED** [review]

**Review by janechu — APPROVED** [review]

**Pull request merged**